modules/hosted-che/partials/ref_about-hosted-che.adoc

```diff
@@ -7,19 +7,19 @@
 = About Eclipse Che hosted by Red Hat
 
 Eclipse Che hosted by Red Hat is an open source product based on link:https://www.eclipse.org/che/[Eclipse {prod-short}] that is running on link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated].
-The new service is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering, and is using link:https://developers.redhat.com/products/codeready-workspaces/overview[CodeReady Workspaces], which is built upon Eclipse Che and is optimized for Red Hat OpenShift and Red Hat Linux.
+The new service is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering, and is using link:https://developers.redhat.com/products/openshift_dev_spaces/overview[{rh-os-devspaces}], which is built upon Eclipse Che and is optimized for Red Hat OpenShift and Red Hat Linux.
 
-== Differences between Eclipse Che and CodeReady Workspaces
+== Differences between Eclipse Che and {rh-os-devspaces}
 
-Red Hat CodeReady Workspaces is the product that is built from the Eclipse Che project. The product is normally two versions behind the project. Red Hat also provides licensing, packaging, and support, so CodeReady Workspaces is considered a more stable product than the upstream Eclipse Che project. See link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.14/html/release_notes_and_known_issues/about-codeready-workspaces_crw[About Red Hat CodeReady Workspaces].
+{rh-os-devspaces} is the product that is built from the Eclipse Che project. The product is normally two versions behind the project. Red Hat also provides licensing, packaging, and support, so {rh-os-devspaces} is considered a more stable product than the upstream Eclipse Che project. See link:https://access.redhat.com/documentation/en-us/red_hat_openshift_devspaces/3.0/html/release_notes_and_known_issues/about-devspaces_crw[About {rh-os-devspaces}].
```

Comment:
Links broken because not published yet: => add them in the .htmltest.yml file, in the IgnoreURLs list
￼
￼
￼
```
IgnoreURLs:
  - https://cse.google.com/cse.js
  - https://marketplace.visualstudio.com
  - https://github.com/eclipse/che-docs/edit/master/
  - https://www.eclipse.org/che/docs/che-7/
  - https://projectlombok.org/
  - https://developers.redhat.com/products/openshift_dev_spaces/overview
  - https://access.redhat.com/documentation/en-us/red_hat_openshift_devspaces/3.0/html/release_notes_and_known_issues/about-devspaces_crw
```